scalalib/src/Lib.scala

```diff
@@ -29,6 +30,7 @@ object Lib {
       deps: IterableOnce[Dep],
       mapDependencies: Option[Dependency => Dependency] = None,
       customizer: Option[coursier.core.Resolution => coursier.core.Resolution] = None,
+      coursierCacheCustomizer: Option[coursier.cache.FileCache[Task] => coursier.cache.FileCache[Task]] = None,
       ctx: Option[Ctx.Log] = None
```

Comment:
Same here, swap these two params to keep binary compatibility (for default params).